.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Lint
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+jobs:
+  lint:
```

Comment:
Here we could add meaningful context and rename the job from `lint` to `Dart` or whatever is being linted.

Reply:
fix: [beebd20](https://github.com/evva-sfw/abrevva-flutter/pull/4/commits/beebd20b665e992166676a79aecc953761b42e12)